_data/software/poppr.yml

```diff
@@ -0,0 +1,42 @@
+---
+schemaVersion: '1.0'
+description:
+  whatItDoes:
+    en: >-
+      Poppr is an R package designed for analysis of populations with mixed
+      modes of sexual and clonal reproduction. 
+    fr: >-
+      Poppr est un package R conçu pour l'analyse des populations à modes mixtes
+      de reproduction sexuelle et clonale.
+category: '43232600'
+homepageURL:
+  en: 'https://grunwaldlab.github.io/poppr/'
+  fr: 'https://grunwaldlab.github.io/poppr/'
+licences:
```

Comment:
This one is weird from the list and the use of `|` https://www.r-project.org/Licenses/

Reply:
It totally is, but nothing stops them from doing OR statements.